pyproject.toml

```diff
@@ -32,7 +32,16 @@ dependencies = [
     'seaborn>=0.11.1',
     'mctx>=0.0.5',
     'networkx>=3.3',
-    'pytest>=6.2.1',
+]
+
+[dependency-groups]
+test = [
+    "ipykernel>=7.1.0",
+    "jupyter>=1.1.1",
+    "nbval>=0.11.0",
+    "pybefit",
```

Comment:
in the same way you're adding pybefit as a dependency to the `test` dependencty group, to be able to run the model-fitting notebook, can you also add the `mediapy` and `pygraphviz` dependencies here? otherwise people won't be able to run the notebook tests without all cells failing on tmaze_demo.ipynb, which requires mediapy (and is a pretty foundational notebook among the new jax suite)